orders/models.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from warehouses.models import Item
+
+
+class Order(models.Model):
+    """
+    Ordered items made by User.
+    """
+    item_id = models.ForeignKey(Item, on_delete=models.CASCADE)
+    date_ordered = models.DateTimeField(auto_now_add=True)
+    order_number = models.CharField(max_length=20)
```

Comment:
Do we really expect alpha numeric characters for the order number?

Reply:
Not really. But usually, order number accepts alpha numeric characters in the system. Should we change it into integer field?